packages/agents/sequencer/src/bindings/server/index.ts

```diff
@@ -111,14 +112,29 @@ export const bindServer = async (): Promise<FastifyInstance> => {
       try {
         const bid = request.body;
         requestContext.transferId = bid.transferId;
-        await storeFastPathData(bid, requestContext);
+        // Validate Input schema
+        const validateInput = ajv.compile(BidSchema);
```

Comment:
There's actually an easier way to do this schema validation directly through Fastify: https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/

We are doing this in other HTTP handlers you can look at.

Reply:
Makes sense. Done